### DISCUSSION TITLE
Uploading ID but receiving "The image you selected does not appear to contain your school name" ERR Help please!

### DISCUSSION THREAD

**Discussion by 1ikeadragon** [community/community#65122] [GitHub Education]
### Hi everyone,

Im uploading my college ID card. It has Validity and my College Name, My Name and details. Still getting the error: "The image you selected does not appear to contain your school name." Any fix? I'd appreciate some assistance. 

What I've tried already? Cropping the ID card so College logo is hidden and its name is clearly visible. Same error.
Really Troubled by this.

**Comment by anishdalvi** [comment]
Try without cropping anything. Like just keep the all contents of ID card visible. **Ensure everything is clearly visible.**
